### PR TITLE
Create new template

### PR DESCRIPTION
**Pull request by adamnitti**
## overall purpose
create new email template

## changes made
- create new email template

### CODE REVIEW

File: csv-bulk-signature/src/email-sig-template.html [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+            </tr>
+          </tbody>
+        </table>
+      </td>
+      <td class="text-container-margin" rowspan={5}>
+        <table cellpadding="0" cellspacing="0" class="table-height">
+          <tbody>
+            <tr>
+              <td class="blue-bold-text">
+                {{fullname}}{{#if title}}, {{credentials}}{{/if}}
```
Comment (adamnitti) [inline csv-bulk-signature/src/email-sig-template.html:21]: handlebars references....

File: csv-bulk-signature/src/main.ts [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 // get csv
 
-const fs = require("fs"); // comes with NODE
-const csv = require("csv-parser"); // install me
-const handlebars = require("handlebars"); // install me
-const zlib = require("zlib"); // install me
+import fs from "fs";
+import handlebars from "handlebars";
+import csv from "csv-parser";
+import zlib from "zlib";
```
Comment (adamnitti) [inline csv-bulk-signature/src/main.ts:7]: updating imports

File: csv-bulk-signature/src/main.ts [hunk 3]
```diff
@@ -50,11 +50,11 @@ const zipWriteStream = fs.createWriteStream("signature-templates.zip");
 zip.pipe(zipWriteStream);
 
 // Add all the HTML files to the zip file
-fs.readdirSync("dist").forEach((file) => {
-  zip.append(fs.createReadStream(`dist/${file}`), { name: file });
-});
+// fs.readdirSync("dist").forEach((file) => {
```
Comment (adamnitti) [inline csv-bulk-signature/src/main.ts:53]: commenting out zip related lines for now

File: csv-bulk-signature/src/styles.css [hunk 4]
```diff
@@ -1,3 +1,38 @@
-.thing {
-  color: red;
+.main-image {
```
Comment (adamnitti) [inline csv-bulk-signature/src/styles.css:1]: went ahead and created preliminary styles.css file

File: csv-bulk-signature/src/email-sig-template.html [hunk 5]
```diff
@@ -1,12 +1,46 @@
...
-    Phone:
-    <a href="tel:{{phone}}">{{phone}}</a>
-  </tr>
-  {{/if}}
-</table>
+<table cellpadding="11" cellspacing="0" class="signature">
+  <tbody>
+    <tr>
+      <td rowspan="5">
+        {/* table containing the logo image */}
```
Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html:5]: remove or replace the JS comment

Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html]: remove or replace the JS comment

File: csv-bulk-signature/src/email-sig-template.html [hunk 6]
```diff
@@ -1,12 +1,46 @@
...
+        <table cellpadding="0" cellspacing="0">
+          <tbody>
+            <tr>
+              <td>
+                <img class="main-image" src={https://temp-ata-signature-assets.s3.amazonaws.com/ATA_LOGO-CPAAdvisor-BT-RGB.png} alt="ata-logo" />
```
Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html]: make the src a handlebars variable `src"{{logoUrl}}"`

File: csv-bulk-signature/src/email-sig-template.html [hunk 7]
```diff
@@ -1,12 +1,46 @@
...
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </td>
+      <td class="text-container-margin" rowspan={5}>
```
Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html]: no curlys `rowspan={5}` -> `rowspan="5"`

File: csv-bulk-signature/src/email-sig-template.html [hunk 8]
```diff
@@ -1,12 +1,46 @@
...
+              <td class="regular-text">{{title}}</td>
+            </tr>
+            <tr>
+              <td class="regular-text">
+                P: {{telephone}}
+                {{#if mobilephone}}M: {{mobilephone}}{{/if}}
+              </td>
+            </tr>
+            <tr>
+              {/* the class 'align-bottom' also controls the height of the row that this cell inhabits */}
```
Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html]: remove or replace the JS comment

**Review by chancesmith — CHANGES_REQUESTED** [review]

File: csv-bulk-signature/package-lock.json [hunk 9]
```diff
@@ -0,0 +1,666 @@
+{
+  "name": "bulk-email-signature-generator",
+  "version": "1.0.0",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {
+    "": {
```
Comment (chancesmith) [inline csv-bulk-signature/package-lock.json]: remove package-lock (already have a yarn-lock)

File: csv-bulk-signature/src/email-sig-template.html [hunk 10]
```diff
@@ -1,12 +1,46 @@
...
+              <td class="align-bottom">
+                {{#if calendly}}
+                <a href={{calendly}}>SCHEDULE A MEETING </a>
```
Comment (chancesmith) [inline csv-bulk-signature/src/email-sig-template.html]: still want to have the double quotes `href="{{calendly}}"

**Review by chancesmith — CHANGES_REQUESTED** [review]

**Review by chancesmith — APPROVED** [review]

**Pull request merged**